extensions/test/NETCore.SetupTests/NETCore.SetupTests.csproj

```diff
@@ -18,10 +18,10 @@
 
   <ItemGroup>
     <ProjectReference Include="../../src/AWSSDK.Extensions.NETCore.Setup/AWSSDK.Extensions.NETCore.Setup.csproj" />
+    <ProjectReference Include="..\..\..\sdk\src\Services\S3\AWSSDK.S3.NetStandard.csproj" />
```

Comment:
nit: Can we use the same slash type in both references?

Reply:
Done